sktime/classification/frequency_based/_rise.py

```diff
@@ -80,36 +130,56 @@ class RandomIntervalSpectralForest(ForestClassifier, BaseClassifier):
     def __init__(
         self,
         n_estimators=200,
-        random_state=None,
         min_interval=16,
         acf_lag=100,
         acf_min_values=4,
+        n_jobs=None,
+        random_state=None,
     ):
         super(RandomIntervalSpectralForest, self).__init__(
             base_estimator=DecisionTreeClassifier(random_state=random_state),
             n_estimators=n_estimators,
         )
         self.n_estimators = n_estimators
-        self.random_state = random_state
         self.min_interval = min_interval
         self.acf_lag = acf_lag
         self.acf_min_values = acf_min_values
+        self.n_jobs = n_jobs
+        self.random_state = random_state
 
         # We need to add is-fitted state when inheriting from scikit-learn
         self._is_fitted = False
 
+    def _make_estimator(self, append=True, random_state=None):
```

Comment:
I'm not sure if we need to add a method for this, does this one come from scikit-learn? They may need more complex functionality in their case because they support warm-starts. 

But in our case a simple function should be enough, as far as I can see, no? 

```python
def _make_estimator(base_estimator, random_state=None):
    estimator = clone(estimator)
    estimator.set_params(**{"random_state": random_state})
    return estimator 
```

And then in `fit`, we can call:
```python
self.estimators_ = [_make_estimator(self.base_estimators, rng.randint(np.iinfo(np.int32).max) for _ in range(self.n_estimators)]
```

I think functions are a lot easier to understand than methods. We may also want to re-use that function in some other classes, I had a related discussion with someone who's working on other algorithms, having it as a stand-alone function makes it easy to re-use it!

Reply:
Thanks! You are right, the parameter `append` is not necessary for the current implementation of `RandomIntervalSpectralForest`. Let's remove it, and keep the code as succinct as possible.